meinberlin/assets/scss/components/_formset.scss

```diff
@@ -0,0 +1,54 @@
+.formset--inline .formset__form {
+    display: inline-block;
+    width: 9em;
+    margin-right: 0.5em;
+    margin-bottom: 0.7em;
+    border-bottom: 1px solid $border-color;
+    padding-right: 1.5em;
+    position: relative;
+
+    input {
+        border: 0;
+        margin-bottom: 0;
+        background-color: $body-bg;
+    }
+}
+
+.formset--inline .formset__form--has-error {
+    border-bottom-color: $brand-danger;
+    border-bottom-width: 5px;
+}
+
+.formset__empty {
```

Comment:
Maybe rename this to `formset__template`? I took some time to understand its purpose.